dags/openshift_nightlies/manifest.yaml

```diff
@@ -78,12 +78,31 @@ platforms:
 
   baremetal: 
     build: ga
-    versions: ["4.10"]
+    versions: ["4.10", 4.11]
     variants: 
     - name: jetski
       config: 
-        install: baremetal/jetski.json
+        install: baremetal/jetski-ovn.json
         benchmarks: baremetal-benchmarks
+    build: dev
+    versions: ["4.10", 4.11]
+    variants:
+    - name: sdn-medium
+      config:
+        install: baremetal/jetski-sdn.json
+        benchmarks: medium-control-plane.json
```

Comment:
We cannot invoke this benchmark json, it has to be from `baremetal-benchmarks` directory and do not change the name of the `.json` file name.

1. `install-bench.json` to be executed after jetski base install
2. `scaleup-bench.json` to be executed after full size 
3. `webfuse-bench.json` to be after day-2 ops, we usually keep all workload in here.

Reply:
@mohit-sheth so currently there isnt a way to support multiple workload variant. it always refers to same `.json` file.